csrankings-p.csv

```diff
@@ -618,6 +618,7 @@ Peter D. Bennett,University of Bristol,http://www.peteinfo.com,t_vvNfcAAAAJ
 Peter D. Grünwald,CWI,http://homepages.cwi.nl/~pdg,wItp6h8AAAAJ
 Peter Damaschke,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/ptr.aspx,NOSCHOLARPAGE
 Peter Danielis,University of Rostock,https://www.vhr.uni-rostock.de/team/vertretungsprofessur/dr-ing-habil-peter-danielis-vertretungsprofessur,XxRH9IoAAAAJ
+Peter Dayan,Max Planck Society,https://www.kyb.tuebingen.mpg.de/computational-neuroscience,_30hSU8AAAAJ

```

Comment:
Not in CS, please clarify.

Reply:
Hi @emeryberger,
Although mainly bases in the MPI for Biological Cybernetics, Peter Dayan also works in fields related to computer science (ML, AI). He is also affiliated to the Department of Computer Science at the University of Tübingen, and is part of the whole AI ecosystems in Tübingen (MPI Campus, Cyber Valley...).
More details here:
https://www.mpg.de/12309370/biological-cybernetics-dayan

Reply:
This doesn't explain how this faculty member meets the inclusion criteria (able to solely advise a PhD _in Computer Science_, for instance).